roc/roc.go

```diff
@@ -1,3 +1,169 @@
+// WARNING: This file has automatically been generated on Sun, 13 Oct 2019 20:21:32 CEST.
+// Code generated by https://git.io/c-for-go. DO NOT EDIT.
+
 package roc
 
-// TODO: replace this with generated bindings
+/*
+#cgo linux CFLAGS: -I"../include"
+#cgo linux LDFLAGS: -lroc
+#include "../include/roc/address.h"
+#include "../include/roc/config.h"
+#include "../include/roc/context.h"
+#include "../include/roc/frame.h"
+#include "../include/roc/log.h"
+#include "../include/roc/platform.h"
+#include "../include/roc/receiver.h"
+#include "../include/roc/sender.h"
+#include <stdlib.h>
+#include "cgo_helpers.h"
+*/
+import "C"
+import "unsafe"
+
+// AddressInit function as declared in roc/address.h:86
+func AddressInit(address *Address, family Family, ip string, port int32) int32 {
+	caddress, _ := address.PassRef()
+	cfamily, _ := (C.roc_family)(family), cgoAllocsUnknown
+	cip, _ := unpackPCharString(ip)
+	cport, _ := (C.int)(port), cgoAllocsUnknown
+	__ret := C.roc_address_init(caddress, cfamily, cip, cport)
+	__v := (int32)(__ret)
+	return __v
+}
+
+// AddressFamily function as declared in roc/address.h:97
+func AddressFamily(address *Address) Family {
+	caddress, _ := address.PassRef()
+	__ret := C.roc_address_family(caddress)
+	__v := (Family)(__ret)
+	return __v
+}
+
+// AddressIp function as declared in roc/address.h:116
+func AddressIp(address *Address, buf *byte, bufsz uint) string {
```

Comment:
This is probably incorrect. `buf` should be a pointer to a buffer of size `bufsz`, and here we're passing a pointer to a single byte. Shouldn't we pass a slice instead?

BTW, a high level wrapper for this function may have much simpler signature:

```
func (addr *Address) IP() string
```